test/e2e/tests/collector_disable_scan/main_test.go

```diff
@@ -28,7 +33,15 @@ func TestMain(m *testing.M) {
 		envfuncs.LoadDockerImageToCluster(util.KindClusterName, util.Image),
 		envfuncs.LoadDockerImageToCluster(util.KindClusterName, util.CollectorImage),
 		envfuncs.LoadDockerImageToCluster(util.KindClusterName, util.NonVulnerableImage),
-		util.DeployEraserHelm(util.EraserNamespace, "--set", `scanner.image.repository=`, "--set", `controllerManager.additionalArgs={--job-cleanup-on-success-delay=1m}`),
+		util.DeployEraserHelm(util.EraserNamespace,
+			"--set", `scanner.image.repository=`,
+			"--set", fmt.Sprintf("eraser.image.repository=%s", eraserImage.Repo),
```

Comment:
should we make these strings (i.e., `eraser.image.repository=`) consts since they are used in every test?